procurement_portal/records/apps.py

```diff
@@ -1,8 +1,21 @@
 from django.apps import AppConfig
+from django.test import Client
+from django.urls import reverse
 
 
 class RecordsConfig(AppConfig):
     name = "procurement_portal.records"
 
     def ready(self):
         import procurement_portal.records.signals  # noqa
+
+        save_xlsx_template()
+
+
+def save_xlsx_template():
+    print("Saving template.xlsx")
+    client = Client()
+    res = client.get(reverse("purchase-records-template-xlsx"))
```

Comment:
why do this via an HTTP request? Couldn't we just call the code this view uses and skip the HTTP?